src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/ConnectivityTest.cs

```diff
@@ -83,6 +83,30 @@ public static void ConnectionTimeoutTestWithThread()
             Assert.True(timeElapsed < threshold);
         }
 
+        [CheckConnStrSetupFact]
+        public static void ProcessIdTest()
+        {
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
```

Comment:
Does the test need the Application Name to be explicitly set in the connection String ?

Reply:
No, Application Name does not need to be explicitly set in the Connection String.
It is being set by default to `Core .Net SqlClient Data Provider` in the [DbConnectionStringCommon.cs](https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System/Data/Common/DbConnectionStringCommon.cs#L234)
This default value is set as part of `SqlConnectionStringBuilder` object in [SqlConnectionStringBuilder.cs](https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnectionStringBuilder.cs#L71)